packages/mobile/src/transactions/feed/TransactionFeed.tsx

```diff
@@ -153,7 +210,7 @@ function TransactionFeed() {
     }
 
     return groupFeedItemsInSections(tokenTransactions)
-  }, [tokenTransactions])
+  }, [JSON.stringify(tokenTransactions.map((tx) => tx.transactionHash))])
```

Comment:
is there an edge case where `tokenTransactions` alone doesn't refresh this `useMemo` value? i think dependency arrays are evaluated every rerender and `JSON.stringify` might cause performance issues if the user does have a large transaction feed + low spec phone?